### PR TITLE
Document new ProgressBar time-based frequency mutators

### LINKED ISSUE #11915: [Console] Add ProgressBar::preventRedrawFasterThan() and forceRedrawSlo…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#26339
| PR author(s) | @ostrolucky
| Merged in    | 4.4

### PR DESCRIPTION
**Pull request by ostrolucky**
Fixes #11915

### CODE REVIEW

File: components/console/helpers/progressbar.rst [hunk 1]
```diff
@@ -50,10 +50,10 @@ you can also set the current progress by calling the
 
     If your platform doesn't support ANSI codes, updates to the progress
     bar are added as new lines. To prevent the output from being flooded,
-    adjust the
-    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::setRedrawFrequency`
-    accordingly. By default, when using a ``max``, the redraw frequency
-    is set to *10%* of your ``max``.
+    adjust the update interval when this happens via these methods
+    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::preventRedrawFasterThan` - write after each x second
```
Comment (OskarStark) [inline components/console/helpers/progressbar.rst]:
```suggestion
    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::preventRedrawFasterThan` - write after each x seconds
```
?

File: components/console/helpers/progressbar.rst [hunk 2]
```diff
@@ -289,17 +289,23 @@ to display it can be customized::
...
+    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::setRedrawFrequency`
+    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::forceRedrawSlowerThan`
     so it updates on only some iterations::
 
         $progressBar = new ProgressBar($output, 50000);
         $progressBar->start();
 
-        // update every 100 iterations
-        $progressBar->setRedrawFrequency(100);
+        // redraw every 100 iterations or every 200ms (in case iterations go too slow)
```
Comment (javiereguiluz) [inline components/console/helpers/progressbar.rst]: the comment says "100 iterations" but the code says "200", right?

Comment (ostrolucky) [reply components/console/helpers/progressbar.rst]: good catch! fixed

**Comment by javiereguiluz** [comment]
Gabriel, thanks for contributing these docs! And congrats on your first Symfony Docs contribution.

File: components/console/helpers/progressbar.rst (hunk 2 above)
Comment (ostrolucky) [reply components/console/helpers/progressbar.rst]: @javiereguiluz I can see this issue is there back, despite it being correct on local branch (setRedrawFrequency(100)). Any idea what happened here?

Comment (javiereguiluz) [reply components/console/helpers/progressbar.rst]: Don't know ... but merged docs use `100` both in the code and the comment: https://github.com/symfony/symfony-docs/blob/4.4/components/console/helpers/progressbar.rst#bar-settings

**Pull request merged**